settings.php

```diff
@@ -34,7 +34,7 @@
 require_once($CFG->dirroot . '/user/profile/lib.php');
 
 $settings->add(new admin_setting_configtext(
-    'facetoface_fromaddress',
+    'facetoface/fromaddress',
```

Comment:
Given we are changing all the configs over, I think we just need an upgrade script to grab all configs in the global table for F2F, and write them across to the new table, otherwise settings will be lost on upgrade.

Reply:
Added an upgrade script: https://github.com/catalyst/moodle-mod_facetoface/pull/122/files#diff-fcb317575e9e6245fdf1590e51a86685b61c2cc9573a9df8034323ccb3ef614fR787